cypress/e2e/search_spec.cy.js

```diff
@@ -12,9 +12,12 @@ describe('search and filter spec', () => {
     cy.get('.product-card')[position]().children().find(`img[alt="${name}"]`)
     .get('.product-name')[position]().contains(name)
     .get('.product-brand')[position]().contains(brand)
-    .get('.product-tags')[position]().contains(tags)
-    //change test for tags after we change UI 
-    //consider adding the product brand and product type to alt text to be something like 'colourpop Lippie Pencil lipstick' 
+    if(tags.length) {
+      cy.get('.tags-container')[position]().children().should('have.length', tags.length)
+      tags.forEach((tag, i) => {
+        cy.get(`.tags-container > :nth-child(${i+1})`)[position]().contains(tag)
+      })
+    }
```

Comment:
Nice way of using forEach to check each tag!

Reply:
thank you!!